spec/swagger.yaml

```diff
@@ -7181,10 +7181,14 @@ definitions:
       stringHashcodes:
         type: array
         description: |-
-            String hashcodes
+          String hashcodes to perform matching with context. Can't be used with the `contentFileUri` parameter.
```

Comment:
Well, it's not that it can't be used with contentFileUri. It's just that if we see both, we pick hashcodes and ignore contentFileUri;

Reply:
@alkulikov it means that we don't declare the behavior when both params are described.